sdk/search/search-documents/tsconfig.json

```diff
@@ -2,7 +2,11 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "outDir": "./dist-esm",
-    "declarationDir": "./types"
+    "declarationDir": "./types",
+    "paths": {
+      "@azure/search-documents": ["./src/index"]
+    }
   },
-  "exclude": ["node_modules", "types", "temp", "browser", "dist", "dist-esm", "./samples/**/*.ts"]
+  "exclude": ["node_modules", "types", "temp", "browser", "dist", "dist-esm"],
```

Comment:
You can remove `"excludes"`, since none of these overlap with the `includes` it has no effect.

Reply:
Done